spec/values/calculation/round.hrx

```diff
@@ -0,0 +1,371 @@
+a {
+  b: 8px;
+}
+
+<===>
+================================================================================
+<===> preserved/NaN/simplify/input.scss
+a {b: round(calc(NaN), calc(NaN))}
+
+<===> preserved/NaN/simplify/output.css
+a {
+  b: calc(NaN);
+}
+
+<===>
+================================================================================
+<===> preserved/operation/two_args/simplify/input.scss
+$c: 10px;
+a {b: round(3.8px + 4px, $c)}
+
+<===> preserved/operation/two_args/simplify/output.css
+a {
+  b: 10px;
+}
+
+<===>
+================================================================================
+<===> preserved/operation/two_args/math/input.scss
+a {b: round(3.4px + 10%, 1px + 4px)}
+
+<===> preserved/operation/two_args/math/output.css
+a {
+  b: round(3.4px + 10%, 5px);
+}
+
+<===>
+================================================================================
+<===> preserved/operation/nearest/math/input.scss
+a {b: round(nearest, 3.8px + 10%, 1.1px + 4px)}
+
+<===> preserved/operation/nearest/math/output.css
+a {
+  b: round(nearest, 3.8px + 10%, 5.1px);
+}
+
+<===>
+================================================================================
+<===> preserved/operation/nearest/simplify/input.scss
+a {b: round(nearest, 115px + 2px, 21px + 4px)}
+
+<===> preserved/operation/nearest/simplify/output.css
+a {
+  b: 125px;
+}
+
+<===>
+================================================================================
+<===> preserved/operation/up/input.scss
+a {b: round(up, 101px, 25px)}
+
+<===> preserved/operation/up/output.css
+a {
+  b: 125px;
+}
+
+<===>
+================================================================================
+<===> preserved/operation/down/input.scss
+a {b: round(down, 122px, 25px)}
+
+<===> preserved/operation/down/output.css
+a {
+  b: 100px;
+}
+
+<===>
+================================================================================
+<===> preserved/operation/to-zero/positive/input.scss
+a {b: round(to-zero, 120px, 25px)}
+
+<===> preserved/operation/to-zero/positive/output.css
+a {
+  b: 100px;
+}
+
+<===>
+================================================================================
+<===> preserved/operation/to-zero/negative/input.scss
+a {b: round(to-zero, -120px, 25px)}
+
+<===> preserved/operation/to-zero/negative/output.css
+a {
+  b: -100px;
+}
+
+<===>
+================================================================================
+<===> preserved/infinity/input.scss
+@use "sass:math" as math;
+a {b: round(up, infinity, infinity)}
+
+<===> preserved/infinity/output.css
+a {
+  b: calc(NaN);
+}
+
+<===>
+================================================================================
+<===> preserved/variable/one_arg/input.scss
+$a: 12.1;
+c {d: round($a)}
+
+<===> preserved/variable/one_arg/output.css
+c {
+  d: 12;
+}
+
+<===>
+================================================================================
+<===> variables/input.scss
```

Comment:
Each of these four should be a separate test.

See https://github.com/sass/sass-spec/blob/main/STYLE_GUIDE.md#do-test-only-one-thing-per-spec

Reply:
👍